src/main/java/hudson/plugins/templateproject/ProxyBuilder.java

```diff
@@ -87,7 +89,16 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
 			if (!builder.perform(build, launcher, listener)) {
```

Comment:
How about putting it before the build performs to get some output before/if it fails?